src/appleseed/foundation/math/scalar.h

```diff
@@ -79,6 +79,7 @@ static const double RcpPiSquare     =  0.1013211836423378;      // 1 / (Pi^2) =
 static const double RcpFourPiSquare =  0.0253302959105844;      // 1 / (4 * Pi^2)
 static const double SqrtTwo         =  1.4142135623730950;      // sqrt(2)
 static const double RcpSqrtTwo      =  0.7071067811865475;      // 1 / sqrt(2) = sqrt(2) / 2
+static const double SqrtThree       =  1.7320508075688772;      // sqrt(3)
```

Comment:
Veeeery minor nitpicking, but this should be 1.7320508075688773 since the next digit is a 9 :)